scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/DefaultModelJobValidatorServiceProvider.java

```diff
@@ -44,6 +44,7 @@
 import org.ow2.proactive.scheduler.common.task.Task;
 import org.ow2.proactive.scheduler.common.task.TaskVariable;
 import org.ow2.proactive.scheduler.common.util.VariableSubstitutor;
+import org.springframework.util.CollectionUtils;
```

Comment:
is it possible to use something else than springframework api here ?

if it only does:

`(variableList != null && !variableList.isempty())`

I would prefer not using it